packages/atlas/src/components/_notifications/NotificationTile/NotificationTile.tsx

```diff
@@ -59,34 +70,38 @@ export const NotificationTile: React.FC<NotificationProps> = ({
 
   if (variant === 'compact') {
     return (
-      <StyledListItem
-        loading={loading}
-        read={read}
-        variant="compact"
-        onClick={onClick}
-        nodeStart={
-          !loading ? (
-            <Avatar size="default" assetUrl={avatarUrl} loading={isLoadingAvatar} />
-          ) : (
-            <SkeletonLoader width={32} height={32} rounded />
-          )
-        }
-        caption={!loading ? `${formattedDate} • ${video.title}` : <SkeletonLoader width="50%" height={19} />}
-        label={
-          !loading ? (
-            <>
-              <Text as="span" variant="t200-strong" secondary>
-                {`${member.handle} `}
-              </Text>
-              <Text as="span" variant="t200-strong">
-                {getNotificationText(notification)}
-              </Text>
-            </>
-          ) : (
-            <SkeletonLoader width="40%" height={20} bottomSpace={2} />
-          )
-        }
-      />
+      <StyledLink to={absoluteRoutes.viewer.video(notification.video.id)}>
+        <StyledListItem
+          loading={loading}
+          read={read}
+          variant="compact"
+          onClick={onClick}
+          nodeStart={
+            !loading ? (
+              <Link to={absoluteRoutes.viewer.member(notification.member.handle)}>
```

Comment:
I'm really not sure about this, 1. AC didn't mention this, 2. when you click on avatar, notification is marked as read but we go to the channel instead of video

Reply:
I just have a short call with Adam and he agrees with you, but he also asked me to consult this with Tomasz(notifications are his work). Unfortunately, Tomasz is unavailable today, so  I'll come back to this PR on Monday when he'll be available.

Reply:
ok, fixed